### PR TITLE
Added support for oom_score_adj

### PR DESCRIPTION
**Pull request by Knetic**
Does what it says on the tin.

Should implement #3565. 
(I made #3589 before, but forgot to sign it. So this ought to work)

### CODE REVIEW

File: compose/config/config_schema_v2.0.json [hunk 1]
```diff
@@ -166,6 +166,7 @@
             }
           ]
         },
+		"oom_score_adj": {"type": "integer", "minimum": -1000, "maximum": 1000},
```
Comment (bfirsh) [inline compose/config/config_schema_v2.0.json]: Spaces, please!

**Comment by bfirsh** [comment]
LGTM, apart from minor nit

**Comment by Knetic** [comment]
@bfirsh Thanks! I fixed the indentation. Not sure how to get Gordon to remove the dco/no tag (which the bot put on when i forgot to sign my commit), but looks like all tests pass.

**Comment by shin-** [comment]
Changes LGTM :+1:

**Comment by aanand** [comment]
LGTM

**Comment by shin-** [comment]
@Knetic do you mind squashing those commits together?

**Comment by Knetic** [comment]
Commits squashed! 

Looks like the Jenkins build failed, but the error message doesn't seem related to the changes in this PR. Not sure if that's required to merge, or how to rerun it.

**Comment by aanand** [comment]
Yeah, they were unrelated. Fixed in master, so I've rerun the build and it's green.

**Pull request merged**